.env.template

```diff
@@ -4,3 +4,6 @@ FLASK_ENV=development
 
 # Change the following values for local development.
 SECRET_KEY=secret-key
+API_KEY =
```

Comment:
Filling in these values should be mentioned in the `README.md` file.

This is a blocker for this passing review.